js/.eslintrc

```diff
@@ -42,7 +42,7 @@
     "no-nested-ternary": "error",
     "eqeqeq": "error",
     "quotes": ["error", "single", { "avoidEscape": true }],
-    "no-unused-vars": ["error", { "argsIgnorePattern": "^(symbol|price|tag|since|limit|params|market|timeframe|api|path|code|currency|response|requestHeaders|requestBody|bidsKey|asksKey|context|config|type|priceKey|amountKey)" }],
+    "no-unused-vars": ["error", { "argsIgnorePattern": "^(symbol|price|tag|since|limit|params|market|timeframe|api|path|code|currency|response|requestHeaders|requestBody|bidsKey|asksKey|amountKey|context|config|type|priceKey|amountKey)" }],
```

Comment:
It's already listed in this line.

Reply:
ohh.. i was confused by old oanda branch, sry.